compiler_gym/envs/llvm/service/passes/config.py

```diff
@@ -173,6 +179,9 @@
     "DAH",  # Bugpoint only.
     "MetaRenamerPass",
     "PAEvalPass",
+    "BarrierNoop",  # Used for debugging pass manager.
+    "StripNonLineTableDebugInfoPass",  # Debug stripping.
+    "StripDeadDebugInfoPass",  # Debug stripping.
```

Comment:
Do we always know that the input binary has no debug info to strip?

Reply:
Good question! Debug info is stripped as part of the [environment setup](https://github.com/facebookresearch/CompilerGym/blob/development/compiler_gym/envs/llvm/service/LlvmEnvironment.cc#L137-L144). I copied this behavior from `opt`